vgen/src/test/java/io/gapi/vgen/testdata/java_main_library.baseline

```diff
@@ -382,23 +381,23 @@ public class LibraryServiceApi implements AutoCloseable {
    *
    * <pre><code>
    * try (LibraryServiceApi libraryServiceApi = LibraryServiceApi.createWithDefaults()) {
-   *   String name = "";
-   *   Shelf callResult = libraryServiceApi.getShelf(name);
+   *   String formattedName = "";
```

Comment:
I noticed here that you aren't making use of the field name patterns to call the appropriate format functions - are you intending on doing that in a separate PR?